extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -95,4 +95,39 @@ export class UserController {
   async whoAmI(): Promise<string> {
     return this.user[securityId];
   }
+  @post('/signup', {
+    responses: {
+      '200': {
+        description: 'User',
+        content: {
+          'application/json': {
+            schema: {
+              'x-ts-type': User,
+            },
+          },
+        },
+      },
+    },
+  })
+  async signUp(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(NewUserRequest, {
+            title: 'NewUser',
+          }),
+        },
+      },
+    })
+    newUserRequest: NewUserRequest,
+  ): Promise<User> {
+    const password = await hash(newUserRequest.password, await genSalt());
```

Comment:
No definition for hash and gen salt

Reply:
They are imported from `bcryptjs`:
https://github.com/strongloop/loopback-next/blob/master/examples/todo-jwt/src/controllers/user.controller.ts#L19